### PR TITLE
docs: Clay Recipes 07 Input Group

### PR DESCRIPTION
**Pull request by pat270**

### CODE REVIEW

File: css/clay_css_2_recipes/07_input_group.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
+# Input Group
+
+This recipe will match Input Group to [Material Design's Filled Text with Leading and Trailing Icons](https://material.io/components/text-fields#anatomy). Clay CSS' Input Groups were modified to accommodate many types of designs. This component is markup heavy, but it allows for more customization versus Bootstrap 4's Input Groups.
```
Comment (wincent) [inline css/clay_css_2_recipes/07_input_group.md:3]: "markup heavy" → "markup-heavy"

**Review by wincent — APPROVED** [review]
lg2m. I have only one nit inline, and it needs a `yarn format`, so I'll just fix the nit, format and merge.

**Comment by wincent** [comment]
So @pat270, this one was `07`. Is there a `06`?

**Comment by pat270** [comment]
Thanks @wincent! `06` is Clay Range, I think I mislabelled the pull.

**Pull request merged**